cmd/update-notifier.go

```diff
@@ -27,6 +27,58 @@ import (
 	"github.com/fatih/color"
 )
 
+// humanizeDuration converts time.Duration to human readable duration.
```

Comment:
The structure is kept previously so that it can be passed around easily just the same argument as why BrowserFlag should be boolean wrapper. I don't know how is the choice driven but i would prefer humanized duration to be a proper parsed value and to have a String() method which makes it print properly in a set format.

All the more this PR has nothing to do with this, i am not why you are bringing unrelated changes in this PR.